src/utils/appearance.ts

```diff
@@ -1,16 +1,69 @@
+import { getAudience } from 'types/component-props/_mixins';
 import { ContentProps, ContentType } from 'types/content-props/_content-common';
+import { getContentLanguages } from './languages';
 
 const contentTypeWithWhiteBackground = [
     ContentType.CurrentTopicPage,
-    ContentType.MainArticle,
     ContentType.FormIntermediateStepPage,
     ContentType.FormsOverview,
+    ContentType.MainArticle,
 ];
 
-export const hasWhiteBackground = (content: ContentProps) => {
+const contentTypesWithWhiteHeader: ReadonlySet<ContentType> = new Set([
+    ContentType.AreaPage,
+    ContentType.CurrentTopicPage,
+    ContentType.FormIntermediateStepPage,
+    ContentType.FormsOverview,
+    ContentType.FrontPage,
+    ContentType.FrontPageNested,
+    ContentType.GenericPage,
+    ContentType.GuidePage,
+    ContentType.OfficeBranchPage,
+    ContentType.OfficeEditorialPage,
+    ContentType.Overview,
+    ContentType.PressLandingPage,
+    ContentType.ProductPage,
+    ContentType.SituationPage,
+    ContentType.ThemedArticlePage,
+]);
+
+export const hasNoTopGap = (content: ContentProps) => {
+    if (
+        content.type === ContentType.FrontPage ||
+        content.type === ContentType.FrontPageNested
+    ) {
+        return (
+            getAudience(content.data.audience) === 'employer' ||
+            getAudience(content.data.audience) === 'provider'
+        );
+    }
+
+    return false;
+};
```

Comment:
Kunne vi se på alternative måter å løse dette på? Hvis disse sidetypene trenger en spesialhåndtering så burde den funksjonaliteten kanskje ligge nærmere de relevante komponentene?

Reply:
Jeg tenkte i utgangspunktet at siden det er PageWrapper som sørger for gap ned til innholdet, så har den også ansvar for å unngå gap ved visse tilfeller. Men jeg har løftet logikken ut til IndexPage som offsetter gap.